### ISSUE TITLE
PsychoPy experiments erroring out due to Pyglet multithreading error

### ISSUE THREAD

**Issue by TEParsons** [pyglet/pyglet#211]
Pyglet seems to be attempting to load wmf codecs and hitting an error due to multithreading, which prevents PsychoPy experiments from running. I get the following output:
```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\site-packages\pyglet\media\codecs\wmf.py", line 834, in __init__
    ole32.CoInitializeEx(None, COINIT_MULTITHREADED)
  File "_ctypes/callproc.c", line 948, in GetResult
OSError: [WinError -2147417850] Cannot change thread mode after it is set

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Program Files\Python38\lib\site-packages\pyglet\media\__init__.py", line 143, in <module>
    add_default_media_codecs()
  File "C:\Program Files\Python38\lib\site-packages\pyglet\media\codecs\__init__.py", line 175, in add_default_media_codecs
    add_decoders(wmf)
  File "C:\Program Files\Python38\lib\site-packages\pyglet\media\codecs\__init__.py", line 128, in add_decoders
    for decoder in module.get_decoders():
  File "C:\Program Files\Python38\lib\site-packages\pyglet\media\codecs\wmf.py", line 890, in get_decoders
    return [WMFDecoder()]
  File "C:\Program Files\Python38\lib\site-packages\pyglet\media\codecs\wmf.py", line 836, in __init__
    warnings.warn('WMF failed to initialize threading:', err.strerror)
TypeError: category must be a Warning subclass, not 'str'
##### Experiment ended. #####
```

Pyglet is version 1.5.5 on Python 3.8.3, PsychoPy is master branch version 5e1334fc

**Comment by benmoran56** [comment]
Looks like the `warnings.warn` call is passed two strings, but it should only be one. I'll get this fixed in the next point release. In the meantime, in codecs/wmf.py, try changing this line:
`warnings.warn('WMF failed to initialize threading:', err.strerror)`
to
`warnings.warn('WMF failed to initialize threading: ' + err.strerror)`

**Comment by benmoran56** [comment]
This should now be fixed in the just-released v1.5.7. I'll close this for now, but please follow up if you still see issues. 

And thanks for opening the ticket.
